docs/networks/l15-testnet.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_position: 2
+---
+
+# L16 Public Testnet
```

Comment:
L16 is desired here?

Reply:
I assumed that I can change things via `edit`